cli/src/lib.rs

```diff
@@ -1625,21 +1660,13 @@ fn start_test_validator(
     flags: Option<Vec<String>>,
     test_log_stdout: bool,
 ) -> Result<Child> {
-    fs::create_dir_all(".anchor")?;
-    let test_ledger_filename = ".anchor/test-ledger";
-    let test_ledger_log_filename = ".anchor/test-ledger-log.txt";
-
-    if Path::new(test_ledger_filename).exists() {
-        fs::remove_dir_all(test_ledger_filename)?;
-    }
-    if Path::new(test_ledger_log_filename).exists() {
```

Comment:
This is removed and not added back, but that's ok since `stream_logs` removes the directory if it exists.